<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>657</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>751</width>
    <height>657</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始游戏</string>
    </property>
    <addaction name="action_newGame1"/>
    <addaction name="action_newGame2"/>
    <addaction name="action_load"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>游戏选项</string>
    </property>
    <addaction name="action_pause"/>
    <addaction name="action_retry"/>
    <addaction name="action_stop"/>
    <addaction name="action_save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_setSpeed"/>
    <addaction name="action_setMusic"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_files"/>
    <addaction name="action_ai"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_newGame1">
   <property name="text">
    <string>单人模式</string>
   </property>
  </action>
  <action name="action_newGame2">
   <property name="text">
    <string>双人模式</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>加载存档</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="text">
    <string>暂停/继续</string>
   </property>
  </action>
  <action name="action_setSize">
   <property name="text">
    <string>场地大小</string>
   </property>
  </action>
  <action name="action_files">
   <property name="text">
    <string>说明</string>
   </property>
  </action>
  <action name="action_ai">
   <property name="text">
    <string>启用ai</string>
   </property>
  </action>
  <action name="action_setMusic">
   <property name="text">
    <string>音乐</string>
   </property>
  </action>
  <action name="action_retry">
   <property name="text">
    <string>重新开始</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="text">
    <string>结束游戏</string>
   </property>
  </action>
  <action name="action_setSpeed">
   <property name="text">
    <string>下落速度</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存游戏</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
